telemetry/statsd/src/main/scala/io/buoyant/telemetry/statsd/Metric.scala

```diff
@@ -0,0 +1,31 @@
+package io.buoyant.telemetry.statsd
+
+import com.timgroup.statsd.StatsDClient
+import com.twitter.finagle.stats.{Counter => FCounter, Stat => FStat}
+import java.util.concurrent.atomic.AtomicInteger
+
+private[statsd] object Metric {
+
+  // counters batch up deltas and flush on send
+  class Counter(statsDClient: StatsDClient, name: String) extends FCounter {
+    val count = new AtomicInteger(0)
+
+    def incr(delta: Int): Unit = {
+      val _ = count.addAndGet(delta)
+    }
+
+    def send: Unit = statsDClient.count(name, count.getAndSet(0))
```

Comment:
You should also note that histograms are windowed internally to the metrics registry and so it doesn't make sense to write histograms any more frequently than the window (1m)